packages/salesforcedx-vscode-soql/src/queryResultsView/media/queryDataView.css

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* 
+This file can be used to override Tabulator CSS
+for custom styling http://tabulator.info/docs/4.8/style
+*/
```

Comment:
Once we get to styling the extension we will likely want to customize webview and the table a bit. This file will be used for that in future work.

Table styles can be overridden:
http://tabulator.info/docs/4.8/style

Reply:
Should this and the other parts of the webview be in soql-tooling? Should we structure this the same way we are handling the builder UI by decoupling it from the extension?

Reply:
After thinking about it, at this point I don't see much benefit to doing that now. The webview currently doesn't make sense as a stand alone app like the LWC app we import and it much much simpler as far as html and js goes. At this point I think the only benefit it organizational. I'm not sure we want to spend the time on moving it over.
Perhaps we can take care of that later as tech debt?
@dehru Thoughts?

Reply:
sounds good.

Reply:
@jgellin-sf Please see the comments above

Reply:
Aha. Thanks.